src/zarr/storage/remote.py

```diff
@@ -131,13 +159,23 @@ def from_url(
         -------
         RemoteStore
         """
-        fs, path = fsspec.url_to_fs(url, **storage_options)
+        opts = storage_options or {}
+        opts = {"asynchronous": True, "use_listings_cache": False, **opts}
```

Comment:
these are two new defaults.

Reply:
Why use_listings_cache=False?

Reply:
I'll be posting an issue here shortly. We could remove this in favor of `refresh=True` in the `ls/find` calls.

Reply:
After looking at this more, I think we can revert this change in favor of #2356.